pyproject.toml

```diff
@@ -70,3 +70,37 @@ module = [
   "psutil",
 ]
 ignore_missing_imports = true
+
+[tool.ruff]
+target-version = "py39"
+# same as black
+line-length = 100
+
+[tool.ruff.lint]
+select = [
+  # flake8 defaults:
+  "F",
+  "E",
+  "W",
+  # flake8-comprehensions
+  "C4",
+  # flake8-2020
+  "YTT",
+  # flake8-implicit-str-concat, but only on a single line (includes bytes)
+  "ISC001",
```

Comment:
I think we also need to add [ISC002](https://docs.astral.sh/ruff/rules/multi-line-implicit-string-concatenation/) to match NIC002 ? If it includes bytes too, I only found one instance of that (in a test), so it might be easier to fix it there